composer.json

```diff
@@ -18,10 +18,10 @@
     "require": {
         "php": "^8.0|^8.1",
```

Comment:
PHP 8 mag je droppen, die is bijna EOL. php 8.2 ook ondersteunen

Reply:
Ik vermoed aangezien we php 8.0 droppen moeten we ook Laravel 8 support droppen (support 8.1/2 niet).